package.json

```diff
@@ -48,6 +48,7 @@
   "devDependencies": {
     "eslint": "7.32.0",
     "husky": "^8.0.1",
-    "systemic": "^4.1.2"
+    "systemic": "^4.1.2",
+    "is-ci": "^3.0.1"
```

Comment:
You need to install it as a production dependency. Otherwise, you'll have the same problem as you have with husky, because in the pipeline it's only installing production dependencies

Reply:
ok, I'll do it, but I think it should be reviewed as we are packaging unnecessary dependencies for production, so the package will be bigger, isn't it?

Reply:
Totally agree, but I haven't found a better solution yet. We'll definitely need to investigate it further in the future